frontend/src/components/common/SideBar.tsx

```diff
@@ -1,11 +1,16 @@
 "use client";
 
-import { useState } from "react";
+import { useState, useEffect } from "react";
 import { Avatar, Button, User, Spacer } from "@nextui-org/react";
 import { usePathname, useRouter } from "next/navigation";
 import classNames from "classnames";
 import Icons from "./Icons";
 import UserDropdown from "./UserDropdown";
+import { useUserContext } from "@/contexts/user-context";
+import userService from "@/helpers/user-service/api-wrapper";
+import Cookies from "js-cookie";
+import { MdOutlineLogin, MdOutlineLogout } from "react-icons/md";
```

Comment:
Please move icons to "Icons" component for consistency.

Reply:
Resolved